pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -262,6 +270,15 @@ stdenv.mkDerivation (fBuildAttrs // {
         targets = fBuildAttrs.bazelTargets;
       }
     }
+    ${
+      bazelCmd {
+        cmd = "run";
+        additionalFlags = fBuildAttrs.bazelRunFlags ++ ["--jobs" "$NIX_BUILD_CORES"];
+        # Bazel run only accepts a single target, but `bazelCmd` expects `targets` to be a list.
+        targets = if fBuildAttrs.bazelRunTarget == null then [] else [fBuildAttrs.bazelRunTarget];
```

Comment:
```suggestion
        targets = lib.optional (fBuildAttrs.bazelRunTarget != null) fBuildAttrs.bazelRunTarget;
```

Reply:
ah good catch!

Reply:
done in https://github.com/NixOS/nixpkgs/pull/237648/commits/9114a2593e9f7a5f215a8692e7f73df903b0e133